packages/api-generator/src/locale/en/v-overlay-location-strategies.json

```diff
@@ -1,7 +1,7 @@
 {
   "props": {
     "locationStrategy": "MISSING DESCRIPTION ([edit in github](https://github.com/vuetifyjs/vuetify/tree/next/packages/api-generator/src/locale/en/v-overlay-location-strategies.json))",
-    "location": "MISSING DESCRIPTION ([edit in github](https://github.com/vuetifyjs/vuetify/tree/next/packages/api-generator/src/locale/en/v-overlay-location-strategies.json))",
+    "location": "Aligns the component towards the `top`, `bottom`, `right`, `left`, can be combined like for example `top right`",
```

Comment:
what about `center`, `start`, and `end`?

Reply:
There are more values than that, which is why we link to the type